docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -75,6 +76,54 @@ i.e. you cannot bind a horizontal pod autoscaler to a replication controller and
 The reason this doesn't work is that when rolling update creates a new replication controller,
 the horizontal pod autoscaler will not be bound to the new replication controller.
 
+## Support for custom metrics
+
+Kubernetes 1.2 brings an alpha support for scaling based on application-specific metrics like QPS(Query per second) or average request latency.
+
+### Prerequisites
+
+The cluster has to be started with `ENABLE_CUSTOM_METRICS` env variable set to `true`.
+
+### Pod configuration
+
+The pods to be scaled must have cAdvisor-specific custom (aka application) metrics endpoint configured. The configuration format is described [here](https://github.com/google/cadvisor/blob/master/docs/application_metrics.md). Kubernetes expects the configuration to 
+  be placed in `definition.json` mounted via a [config map](configmap.md) in `/etc/custom-metrics`. A sample config map may look like this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cm-config
+data:
+  definition.json: "{\"endpoint\" : \"http://localhost:8080/metrics\"}"
+``` 
+
+**Warning**
+Due to the way cAdvisor currently works `localhost` refers to the node itself, not to the running pod. Thus the appropriate container in the pod must ask for a node port. Example:
```

Comment:
@janetkuo I remember that you did a PR that ensured "pod" was capitalized to "Pod" -- is that applicable here?